lametro/views.py

```diff
@@ -283,8 +284,44 @@ def get_context_data(self, **kwargs):
 
         context["USING_ECOMMENT"] = settings.USING_ECOMMENT
 
+        # Only provide notices for this event's public comment setting
+        if event.accepts_live_comment:
+            notices_by_comment = EventNotice.objects.filter(
+                comment_conditions__contains=["accepts_live_comment"]
+            )
+        elif event.accepts_public_comment:
+            notices_by_comment = EventNotice.objects.filter(
+                comment_conditions__contains=["accepts_comment"]
+            )
+        else:  # Events that do not accept public comment at all
+            notices_by_comment = EventNotice.objects.filter(
+                comment_conditions__contains=["accepts_no_comment"]
+            )
+
+        # Further filter notices by event broadcast status
+        context["event_status"] = self.get_event_status(event)
+        for status in ("future", "upcoming", "ongoing", "concluded"):
+            if context["event_status"] == status:
+                event_notices = notices_by_comment.filter(
+                    broadcast_conditions__contains=[status]
+                )
+                break
+
+        context["event_notices"] = event_notices
         return context
 
+    def get_event_status(self, event):
+        # Returns the event's broadcast status as a string
+
+        if event.is_upcoming:
+            return "upcoming"
+        elif event.is_ongoing:
+            return "ongoing"
+        elif event.has_passed:
+            return "concluded"
+        else:
+            return "future"
```

Comment:
I have this method in the view returning the matching string per event status.

The LAMetroEvent model has something similar in the `display_status` property, but the statuses that it returns are just different enough that I decided against using it. It returns a `cancelled` status which we're not using and lumps upcoming and future events into the `upcoming` status. I could change the latter bit to fit our needs if we want to! Provided that it's okay to have a new Future label on events